### ISSUE TITLE
usr/bin/env: node: No such file or directory 

### ISSUE THREAD

**Issue by regenrek** [cawa-93/dts-for-context-bridge#2]
Hi,

I've noticed that the lib breaks when you're (what I think) using [`nvm`](https://github.com/nvm-sh/nvm). It seems that `/usr/bin/env node` is not found. within `node_modules`. I can't 100% figure out whats the issue.

I run into this issue on [vite electron builder](https://github.com/cawa-93/vite-electron-builder) repository.
```

> pretypecheck-renderer
> dts-cb -i packages/preload/src/**/*.ts -o packages/preload/exposedInMainWorld.d.ts

env: node\r: No such file or directory
```

**Comment by cawa-93** [comment]
Does't it work if you run command directly (with quotes) ?
```
dts-cb -i "packages/preload/src/**/*.ts" -o "packages/preload/exposedInMainWorld.d.ts"
```

**Comment by regenrek** [comment]
Can I install it globally? 

```
./node_modules/dts-for-context-bridge/bin/index.js -i "packages/preload/src/**/*.ts" -o "packages/preload/exposedInMainWorld.d.ts"

env: node\r: No such file or directory
```

**Comment by cawa-93** [comment]
@regenrek What is your environment?

**Comment by regenrek** [comment]
Pretty default:

* https://github.com/nvm-sh/nvm
* lts/gallium v16.3
* mac os catalina

The vite electron builder works fine. I just struggle with the `dts-cb`. 

```
/usr/bin/env node
```
works out of the box

**Comment by cawa-93** [comment]
I guess the problem is in the end-of-line characters.  Could you check the version from the last commit?

**Comment by regenrek** [comment]
Perfect. This commit has solved the issue

**Comment by cawa-93** [comment]
@regenrek fix released in v0.7.1
